### DISCUSSION TITLE
Sahi with yolo11

### DISCUSSION THREAD

**Discussion by AJITKUMAR130012** [obss/sahi#1093] [Q&A] [answered]
I have trained the yolo11
For the dependency installation, i have used the following command by creating the enviroment through the virtualenv:
**pip install sahi**
**conda install pytorch=1.13.1 torchvision=0.14.1 pytorch-cuda=11.7 -c pytorch -c nvidia**
**pip install ultralytics**

To run the inference, I have used the below command:

sahi predict --model_path yolo11_face_best.pt --model_type ultralytics --source "video.mp4"
( without using the command - pip install yolov5)
Got below error:
![image](https://github.com/user-attachments/assets/16c9e84d-23bc-4908-97cf-b937c4a2992f)

Then i tried the below command by installing yolov5:
**pip install yolov5==7.0.13**
**sahi predict --model_path yolo11_face_best.pt --model_type yolov5 --source "video.mp4"**
Still got the error:
![image](https://github.com/user-attachments/assets/249178b1-3614-4807-9de4-8ed5ddb0b985)

Please suggest me, what command need to use for the yolo11

**Comment by SDIX-7** [comment]
`sahi predict --model_path yolo11_face_best.pt --model_type yolov8 --source "video.mp4"`
YOLOv11 is an enhancement based on YOLOv8, which means you can do almost all of YOLOv8's options for YOLOv11 without modifying the code

**Comment by fcakyon** [comment, marked as answer]
Hey @AJITKUMAR130012 @SDIX-7, Ultralytics YOLOv5U/YOLOv8/YOLO11 OBB models are supported with latest sahi release: https://github.com/obss/sahi/releases/tag/0.11.20

Usage is similar to Ultralytics det/seg models: https://github.com/obss/sahi/blob/main/demo/inference_for_ultralytics.ipynb

This now works: `sahi predict --model_path yolo11_face_best.pt --model_type ultralytics --source "video.mp4"`

**Comment by bshakya77** [comment]
Hello @fcakyon,

Does `predict_evaluate_analyse.py` supports "ultralytics" or "yolo" model type. Or, it is just for "mmdet" model type? I see it is used with TOOD and FCOS model.

Thank you.

Regards,
Bijay
